xmtp_mls/src/storage/encrypted_store/group_message.rs

```diff
@@ -140,6 +141,14 @@ impl DbConnection {
             query = query.filter(dsl::delivery_status.eq(status));
         }
 
+        if let Some(dir) = direction {
+            query = match dir {
+                SortDirection::Ascending => query.order(dsl::sent_at_ns.asc()),
+                SortDirection::Descending => query.order(dsl::sent_at_ns.desc()),
+                SortDirection::Unspecified => query,
```

Comment:
Do we need Unspecified if the direction argument is optional?

Reply:
Ahhh. Because you're taking the proto enum, which always comes with Unspecified.

Could make a specific enum for the DB that only has two options and convert Unspecified -> None

Reply:
It is like putting a hat on a hat. What would you think of making direction non-optional in the args?

Reply:
Since its defined in the proto it is required here, or using `_ =>` but doesn't matter much in this case